Microsoft.OData.AzureFunctions/ODataValueProvider.cs

```diff
@@ -27,15 +25,16 @@ public async Task<object> GetValueAsync()
                 IEdmModel model = this.modelProvider.GetEdmModel();
 
                 // TODO: Add routePrefix to constructor to be passed from the function
-                return ODataBindingHelper.BuildODataQueryOptions(this.request, model, this.clrType, this.routePrefix);
+                Type clrType = typeof(T).GenericTypeArguments.First();
```

Comment:
```suggestion
                Type clrType = this.Type.GenericTypeArguments.First();
```

Reply:
Why?

Reply:
Personal preference because of line 37 `public Type Type => typeof(T);`